### DISCUSSION TITLE
join a grid of images together

### DISCUSSION THREAD

**Discussion by soorsoor** [ImageMagick/ImageMagick#3781] [Help]
hi 
i have a 32x64 collection of png images that were cropped to a grid from a large original image 
they are named in order of their location(coordinates) in the original image :
tx_0_0 , tx_0_1 , tx_0_2 , tx_1_0 , tx_1_1 , tx_1_2 , ...
i want to simply join them together to get a one large image
i figured i should use the montage command, but i don't know how to iterate through the names or extract the x,y from the files' names.
i'm using the latest windows version.
appreciate your help :)

**Comment by fmw42** [comment]
What platform/OS and what version of ImageMagick?  You might also consider doing a double loop and appending all images in each row. Then append the rows.  Typically images are listed in alphabetic order in your file system. Check the order. It might already be correct. Montage needs all the images listed left to right, top to bottom. If your images are not in the correct order, they might become so if you use leading zeros. So rather than 0, use 00 and rather than  1, use 01 etc. For 10 and up to 63, just use 10 up to 63.

**Reply by soorsoor** [reply]
windows10  imagemagick7
thanks but im not good at shell code, im stuck at making the loop with the inputs
